packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
+        assert(
+          Nat(delaySecs) >= 0,
+          details`createNotifier's first parameter must be a non-negative integer. ${delaySecs}`,
+        );
+        assert(
+          Nat(interval),
+          details`createNotifier's second parameter must be an integer, ${interval}`,
+        );
+
+        const index = D(timerNode).createRepeater(delaySecs, interval);
+        console.log(`TIMER: index: ${index}`);
```

Comment:
Are these `console.log` calls in the production code? (also line 60 below) Or are they debugging vestiges? I notice that the other console methods have no such `console` calls.

Reply:
I think we should remove the `console.log`s